src/components/buttons/z-chip/index.tsx

```diff
@@ -56,23 +58,28 @@ export class ZChip {
             />
           )}
           <slot />
-          <z-icon
+          <button
+            type="button"
             tabIndex={this.disabled ? -1 : 0}
```

Comment:
Questa cosa serve? Non è sufficiente mettere il bottone a disabled quando serve, e rimuovere completamente il tabindex?

Reply:
Da un punto di vista funzionale funziona in entrambe i casi,  ma per coerenza pareva brutto avere un tabindex=0 quando l'elemento non è "tabbabile".